src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components/macro'
+import React from 'react'
+import SmallCard from '../SmallCard/SmallCard'
+
+export default function HomePage({ cards, setCards }) {
+  return (
+    <HomePageLayout>
+      {cards.map(card => (
+        <SmallCard
+          name={card.name}
+          store={card.store}
+          price={card.price}
+          date={card.date}
+          clothingType={card.clothingType}
+          images={card.images}
+          cards={cards}
+          setCards={() => setCards()}
+        />
+      ))}
+    </HomePageLayout>
+  )
+}
+const HomePageLayout = styled.div`
```

Comment:
Suggestion:
If I have understood your code correctly, you could use `main` instead of `div` for more semantic HTML